packages/server/src/constants.ts

```diff
@@ -57,6 +57,14 @@ export const ERRORS = {
     status_code: 410,
     body: 'The file for this url no longer exists\n',
   },
+  ERR_SIZE_EXCEEDED: {
+    status_code: 413,
+    body: "upload's size exceeded\n",
+  },
+  ERR_MAX_SIZE_EXCEEDED: {
+    status_code: 413,
+    body: 'Maximum size exceeded\n',
+  },
```

Comment:
Can this be the same error? They seem very similar.

Reply:
it can be yes, for some reason tusd seems to differentiate them

Reply:
MaxSizeExceeded: https://github.com/tus/tusd/blob/1a43e26f16f43bed5dd2219c27e6eb14c125fb03/pkg/handler/unrouted_handler.go#L31

SizeExceeded: https://github.com/tus/tusd/blob/1a43e26f16f43bed5dd2219c27e6eb14c125fb03/pkg/handler/unrouted_handler.go#L39

Technically i think they are 2 different errors:

- MaxSizeExceeded is returned when the body size is greater than the defined server max size
- SizeExceeded is returned when trying to upload more bytes than the declared size of the file 

I would personally keep them separate too